microsoftgraph/microsoft-graph-explorer
# Task/cred scan secret

## Overview

Substitutes dummy password with a random and unique password.
### Demo

N/A

### Notes

The dummy password was flagged by a tool as a security risk that can be taken advantage of by bad actors. Graph Explorer now substitutes the dummy password with a random and unique password.

## Testing Instructions

* Log In to Graph Explorer
* Click on Create User sample
* Observe that the password is  a `random string`
